src/extension/webView.ts

```diff
@@ -106,20 +106,24 @@ export async function createRepository(context: vscode.ExtensionContext) {
           }
 
           // Go to target folder & create app folder
-          const createAppFolder = `cd '${options.targetFolder}' && mkdir '${options.name}' && cd '${options.name}'`;
+          const createAppFolder = `mkdir '${options.targetFolder}${options.name}' -p && cd '${options.targetFolder}${options.name}'`;
```

Comment:
I think it must be `mkdir -p ...` because on MacOS / Unix the argument needs to be beforehand.